app/js/components/manageApps/ConfirmationModal.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Modal,
+  ModalHeader,
+  ModalTitle,
+  ModalClose,
+  ModalBody,
+  ModalFooter
+} from 'react-modal-bootstrap';
+
+export default class ConfirmationModal extends Component {
```

Comment:
Since there's no state change in this component, a stateless functional component should be used instead